enterprise/api_client/enterprise.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
```

Comment:
This was originally from #169, but I needed it for this so copied it here and modified it to be generic + have the needed additions for programs.